main_test.go

```diff
@@ -16,34 +16,32 @@ func TestFindRoot(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	expect := filepath.Join(wd, "_testdata", "rootfind")
-
-	got1, err := findProjectRoot(expect)
+	want := filepath.Join(wd, "_testdata", "rootfind")
+	got1, err := findProjectRoot(want)
 	if err != nil {
 		t.Errorf("Unexpected error while finding root: %s", err)
-	} else if expect != got1 {
-		t.Errorf("findProjectRoot directly on root dir should have found %s, got %s", expect, got1)
+	} else if want != got1 {
```

Comment:
is preferring "want" to "expect" a standard thing? I don't see it mentioned explicitly in the [code review docs](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures) (though the table test prop obviously is named `want`).